crux/src/Crux.hs

```diff
@@ -605,8 +605,7 @@ printFailedGoals opts (CruxSimulationResult _cmpl allGls)
     = sayFail "Crux" (show $ vcat [ "Found counterexample for verification goal", ex ])
 
     | otherwise
-    = sayFail "Crux" (show $ vcat [ "Failed to prove verification goal", ex, ppSimError err, text msg])
-
+    = sayFail "Crux" (show $ vcat [ "Failed to prove verification goal", ex ])
```

Comment:
Is this change because the extra information above isn't really user-comprehensible?

Reply:
No, the change is because we now pass "unknown" goals into the same explanation callback as "disproved" goals.  This gives clients more control over what gets printed.

Reply:
Oh, got it. That makes sense.